src/webapi/request_animation_frame_handle.rs

```diff
@@ -0,0 +1,23 @@
+/// Struct representing an animation frame request
+/// Use webapi::window::request_animation_frame to create this
```

Comment:
`Struct representing an animation frame request` -> `A handle to a pending animation frame request.`, or something like that.

If you want to point to the function which creates it then you can link to it properly (the docs are markdown; look at other doc comments in other files for examples on how to do it)